irods/manager/user_manager.py

```diff
@@ -57,28 +57,25 @@ def get(self, user_name, user_zone=""):
             raise UserDoesNotExist()
         return iRODSUser(self, result)
 
-    def create_remote(self, user_name : str, user_zone : str):
+    def create_remote(self, user_name:str, user_zone:str):
         """
         Create an entry in the local catalog for a remote user.  The user_type will be 'rodsuser'.
         """
         if user_zone in (self.sess.zone,""):
-            raise RuntimeError(f"Parameter [{user_zone = }] must be a remote zone.")
+            raise ValueError(f"Parameter [{user_zone = }] must be a remote zone.")
         return self.create_with_password(user_name, password='', user_zone=user_zone)
 
-    def create_with_password(self, user_name, password, user_zone=""):
+    def create_with_password(self, user_name:str, password:str, user_zone:str=""):
         """This method can be used by a groupadmin to initialize the password field while creating the new user.
         (This is necessary since group administrators may not change the password of an existing user.)
         """
 
-        if "#" not in user_name[:-1]:
-            _user_name = user_name + ("" if not user_zone else f"#{user_zone}")
-        else:
-            _user_name = user_name
-            user_name, user_zone = user_name.split("#",1)
+        if user_zone and password:
```

Comment:
I think we want to disallow this only if `user_zone` is not referring to the local zone (whether explicitly or by being empty), right?

Reply:
is your case handled below on 78?

Reply:
I'm thinking of this case:
```python
groupadmin.users.create_with_password("newUser", "newPassword", user_zone="tempZone")
```
We provide a `password` and `zone_name`, but it's the local zone. I think that should be allowed. But this...
```python
groupadmin.users.create_with_password("newUser", "newPassword", user_zone="otherZone")
```
...would not be allowed, because it's providing a `password` and a `zone_name` and the `zone_name` is not the local zone.

Reply:
Agreed.

Reply:
Fixed now, I think. rsolving